### ISSUE TITLE
not found interface discard in ports table

### ISSUE THREAD

**Issue by kaoru290** [librenms/librenms#2594]
where `ifInDiscard` or `ifOutDiscard` ?
my ports table below.
```
mysql> show columns from ports;
+-------------------------+--------------+------+-----+-------------------+-----------------------------+
| Field                   | Type         | Null | Key | Default           | Extra                       |
+-------------------------+--------------+------+-----+-------------------+-----------------------------+
| port_id                 | int(11)      | NO   | PRI | NULL              | auto_increment              |
| device_id               | int(11)      | NO   | MUL | 0                 |                             |
| port_descr_type         | varchar(255) | YES  |     | NULL              |                             |
| port_descr_descr        | varchar(255) | YES  |     | NULL              |                             |
| port_descr_circuit      | varchar(255) | YES  |     | NULL              |                             |
| port_descr_speed        | varchar(32)  | YES  |     | NULL              |                             |
| port_descr_notes        | varchar(255) | YES  |     | NULL              |                             |
| ifDescr                 | varchar(255) | YES  | MUL | NULL              |                             |
| ifName                  | varchar(64)  | YES  |     | NULL              |                             |
| portName                | varchar(128) | YES  |     | NULL              |                             |
| ifIndex                 | int(11)      | YES  |     | 0                 |                             |
| ifSpeed                 | bigint(20)   | YES  |     | NULL              |                             |
| ifConnectorPresent      | varchar(12)  | YES  |     | NULL              |                             |
| ifPromiscuousMode       | varchar(12)  | YES  |     | NULL              |                             |
| ifHighSpeed             | int(11)      | YES  |     | NULL              |                             |
| ifOperStatus            | varchar(16)  | YES  |     | NULL              |                             |
| ifAdminStatus           | varchar(16)  | YES  |     | NULL              |                             |
| ifDuplex                | varchar(12)  | YES  |     | NULL              |                             |
| ifMtu                   | int(11)      | YES  |     | NULL              |                             |
| ifType                  | text         | YES  |     | NULL              |                             |
| ifAlias                 | text         | YES  |     | NULL              |                             |
| ifPhysAddress           | text         | YES  |     | NULL              |                             |
| ifHardType              | varchar(64)  | YES  |     | NULL              |                             |
| ifLastChange            | timestamp    | NO   |     | CURRENT_TIMESTAMP | on update CURRENT_TIMESTAMP |
| ifVlan                  | varchar(8)   | NO   |     |                   |                             |
| ifTrunk                 | varchar(8)   | YES  |     |                   |                             |
| ifVrf                   | int(11)      | NO   |     | 0                 |                             |
| counter_in              | int(11)      | YES  |     | NULL              |                             |
| counter_out             | int(11)      | YES  |     | NULL              |                             |
| ignore                  | tinyint(1)   | NO   |     | 0                 |                             |
| disabled                | tinyint(1)   | NO   |     | 0                 |                             |
| detailed                | tinyint(1)   | NO   |     | 0                 |                             |
| deleted                 | tinyint(1)   | NO   |     | 0                 |                             |
| pagpOperationMode       | varchar(32)  | YES  |     | NULL              |                             |
| pagpPortState           | varchar(16)  | YES  |     | NULL              |                             |
| pagpPartnerDeviceId     | varchar(48)  | YES  |     | NULL              |                             |
| pagpPartnerLearnMethod  | varchar(16)  | YES  |     | NULL              |                             |
| pagpPartnerIfIndex      | int(11)      | YES  |     | NULL              |                             |
| pagpPartnerGroupIfIndex | int(11)      | YES  |     | NULL              |                             |
| pagpPartnerDeviceName   | varchar(128) | YES  |     | NULL              |                             |
| pagpEthcOperationMode   | varchar(16)  | YES  |     | NULL              |                             |
| pagpDeviceId            | varchar(48)  | YES  |     | NULL              |                             |
| pagpGroupIfIndex        | int(11)      | YES  |     | NULL              |                             |
| ifInUcastPkts           | bigint(20)   | YES  |     | NULL              |                             |
| ifInUcastPkts_prev      | bigint(20)   | YES  |     | NULL              |                             |
| ifInUcastPkts_delta     | bigint(20)   | YES  |     | NULL              |                             |
| ifInUcastPkts_rate      | int(11)      | YES  |     | NULL              |                             |
| ifOutUcastPkts          | bigint(20)   | YES  |     | NULL              |                             |
| ifOutUcastPkts_prev     | bigint(20)   | YES  |     | NULL              |                             |
| ifOutUcastPkts_delta    | bigint(20)   | YES  |     | NULL              |                             |
| ifOutUcastPkts_rate     | int(11)      | YES  |     | NULL              |                             |
| ifInErrors              | bigint(20)   | YES  |     | NULL              |                             |
| ifInErrors_prev         | bigint(20)   | YES  |     | NULL              |                             |
| ifInErrors_delta        | bigint(20)   | YES  |     | NULL              |                             |
| ifInErrors_rate         | int(11)      | YES  |     | NULL              |                             |
| ifOutErrors             | bigint(20)   | YES  |     | NULL              |                             |
| ifOutErrors_prev        | bigint(20)   | YES  |     | NULL              |                             |
| ifOutErrors_delta       | bigint(20)   | YES  |     | NULL              |                             |
| ifOutErrors_rate        | int(11)      | YES  |     | NULL              |                             |
| ifInOctets              | bigint(20)   | YES  |     | NULL              |                             |
| ifInOctets_prev         | bigint(20)   | YES  |     | NULL              |                             |
| ifInOctets_delta        | bigint(20)   | YES  |     | NULL              |                             |
| ifInOctets_rate         | int(11)      | YES  |     | NULL              |                             |
| ifOutOctets             | bigint(20)   | YES  |     | NULL              |                             |
| ifOutOctets_prev        | bigint(20)   | YES  |     | NULL              |                             |
| ifOutOctets_delta       | bigint(20)   | YES  |     | NULL              |                             |
| ifOutOctets_rate        | int(11)      | YES  |     | NULL              |                             |
| poll_time               | int(11)      | YES  |     | NULL              |                             |
| poll_prev               | int(11)      | YES  |     | NULL              |                             |
| poll_period             | int(11)      | YES  |     | NULL              |                             |
+-------------------------+--------------+------+-----+-------------------+-----------------------------+
70 rows in set (0.00 sec)
```

**Comment by f0o** [comment]
We only have that field in the RRD's it seems

**Comment by kaoru290** [comment]
thanks.
i want to alert setting of interface discard.
can i config it?

**Comment by laf** [comment]
We merged a PR (#2542) today that adds this info to a new table: ports_statistics so you can use that to alert on once your install is updated.

**Comment by kaoru290** [comment]
@laf i found it!
thank you!
